Tasks/AppCenterDistributeV3/task.loc.json

```diff
@@ -110,7 +111,7 @@
       "groupName": "symbols",
       "required": false,
       "helpMarkDown": "ms-resource:loc.input.help.mappingTxtPath",
-      "visibleRule": "symbolsType = AndroidJava"
+      "visibleRule": "symbolsType = AndroidProguard"
```

Comment:
does this have any impact on existing definitions (renaming `AndroidJava` to `AndroidProguard`)?

Reply:
The `AndroidJava` symbolsType was deprecated a long time ago or never published. It was not public in AppCenterDistributeV3 so there shouldn't be any impact.